Kernel/Devices/Audio/IntelHDA/Stream.h

```diff
@@ -49,6 +54,24 @@ class Stream {
     enum StreamControlFlag : u32 {
         StreamReset = 1u << 0,
         StreamRun = 1u << 1,
+        InterruptOnCompletionEnable = 1u << 2,
+    };
+
+    // 3.3.36 : Input/Output/Bidirectional Stream Descriptor Status
+    enum StreamStatusFlag : u8 {
+        BufferCompletionInterruptStatus = 1u << 2,
+    };
+
+    // 3.6.3: Buffer Descriptor List Entry
+    enum BufferDescriptorEntryFlag : u32 {
```

Comment:
Those should both be enum classes as per standard coding style

Reply:
Can you point me to the coding style where this is mentioned?

Reply:
I know it's a best practice in general, but I don't think it's explicitly stated in our coding style. That being said, these new enums reflect the style in which all the other enums in the vicinity were coded and I'd rather not diverge from them or refactor them all in this PR. Additionally, the code would be littered with `to_underlying(..)` everywhere or I would need to have helper methods to keep the code readable - and that's what I was going for.

Reply:
Okay, I guess it's fine in this context. For reference, @ldm5180 did a bunch of work converting enums to enum classes, and as a TL;DR even though to_underlying is necessary for many places, enum classes provide much-improved semantics over enums which are technically just C compatibility.

Reply:
Yeah, a refactor is probably due for these enums, it's just not a lot of fun to do :^)